state_machine/plugs/flight/lib/pycubed.py

```diff
@@ -78,7 +77,8 @@ def __init__(self):
             'GPS':    False,
             'WDT':    False,
             'Sun':    False,
-            'Coils':  False
+            'Coils':  False,
+            'BurnWire': False
```

Comment:
This is a bit pedantic but generally speaking to reduce the future number of lines changes in commit we want to put a comma even on the last entry to the dictionary:
ie ```
a = {
    "1": True,
    "2": False,
}
If the previous author had stuck to this convention adding the `'BurnWire': False,` would have only been one insertion (rather than 1 deletion and two insertions)